src/gui_common/CustomRichTextLabel.cs

```diff
@@ -73,12 +77,20 @@ public enum ThriveBbCode
         }
     }
 
+    [Export]
+    public bool EnableTooltipsForMetaTags { get; set; } = true;
+
     public override void _Ready()
     {
         // Make sure bbcode is enabled
         BbcodeEnabled = true;
 
+        speciesPreviewTooltipScene = GD.Load<PackedScene>(
```

Comment:
There can be a ton of rich text labels. I think it'd be better to have this be a lazy static property or a property on tooltip manager to keep a reference to this scene and not need to attempt to load it for each created text control.